File: 25 Coding Challenge I/JavaScript/Challenge11.js
// Challenge 11: Calculate the average of the numbers in an array of numbers

function averageArray(arr){
    let sum = 0;

    for (let i = 0; i < arr.length; i++){
        sum += arr[i];
    }

    return sum / arr.length;
}

// Driver Method
let arr = [1, 3, 9, 15, 90];
console.log("Average " + averageArray(arr));

File: 50 Coding Challenge Part I/dart/lib/challenge11.dart
/*  
  Coding Challenge 11: Calculate the average of the numbers in an array of numbers
  Example array: [1, 3, 9, 15, 90]
  Expected output: 23.6

  @author: mofirojean
*/

import 'package:dart/challenge10.dart';

double averageArray(List<int> arr) {
  return sumArray(arr) / arr.length;
}
